### PR TITLE
704 document quick start removal

### PR DESCRIPTION
**Pull request by uberskigeek**
### Checklist:

- [x] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [x] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [x] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md#stack-structure).

### Modifying an existing stack:

- [x] Updated the stack version in `stack.yaml`

### Contributing a new stack:

- Describe how application dependencies are managed:

- Explain how Appsody file watcher is utilized:

- Describe other Appsody environment variables defined by the stack image:

- Describe any limitations and known issues:

### CODE REVIEW

**Comment by uberskigeek** [comment]
#704

File: incubator/java-microprofile/README.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ OpenAPI endpoints:
 - http://localhost:9080/openapi (the RESTful APIs of the inventory service)
 - http://localhost:9080/openapi/ui (Swagger UI of the deployed APIs)
 
+### Quick start security
```
Comment (scottkurz) [inline incubator/java-microprofile/README.md]: This is good.. but did you look at the java-openliberty stack ?   Worth using the same doc?   (Not saying that one's better).

File: incubator/java-microprofile/image/project/Dockerfile [hunk 2]
```diff
@@ -23,6 +23,7 @@ RUN cd /project/user-app && mvn -B liberty:install-server
 # Remove quick-start-security.xml since it is only needed during local development.
 COPY ./user-app/src /project/user-app/src
 RUN cd /project/user-app && \
+    echo "QUICK START SECURITY IS NOT SECURE FOR PRODUCTION ENVIRONMENTS. IT IS BEING REMOVED" \
```
Comment (scottkurz) [inline incubator/java-microprofile/image/project/Dockerfile]: Should we add this too to java-openliberty ?

Comment (uberskigeek) [reply incubator/java-microprofile/image/project/Dockerfile:26]: Ok I'll add it there.

File: incubator/java-microprofile/README.md [hunk 3]
```diff
@@ -44,6 +44,13 @@ OpenAPI endpoints:
 - http://localhost:9080/openapi (the RESTful APIs of the inventory service)
 - http://localhost:9080/openapi/ui (Swagger UI of the deployed APIs)
 
+### Config dropin: **quick-start-security.xml**
+
+The metrics endpoint is secured with a userid and password enabled through the config dropin included in the default template at path:
+**src/main/liberty/config/configDropins/defaults/quick-start-security.xml**.
+
+In order to lock down the production image built via `appsody build` this file is deleted during the Docker build of your application production image.  (The same file would be deleted if you happened to create your own file at this location as well).
+
```
Comment (Emily-Jiang) [inline incubator/java-microprofile/README.md:53]: An instruction should be provided on how to provide userid and password to access the metrics endpoint.

Comment (uberskigeek) [reply incubator/java-microprofile/README.md:53]: This will be addressed by #750

**Comment by scottkurz** [comment]
@skoh7645 @Kamran64 we have one more when you get a chance.  

This would be:  

**java-microprofile:** 
  * echo informational message about quick-start-security.xml during `appsody build`

**java-openliberty:** 
 * echo informational message about quick-start-security.xml during `appsody build`
 * Update stack description.

Thank you again.

**Comment by uberskigeek** [comment]
Fixes #704

**Comment by skoh7645** [comment]
@uberskigeek @scottkurz This PR LGTM, but we need to split this PR to do releases for both the `java-microprofile` and `java-openliberty` stacks

**Pull request merged**